components/smalltalk/squeak5/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,50 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/squeak
+file path=usr/doc/squeak/COPYING.gz
+file path=usr/doc/squeak/COPYRIGHT.gz
+file path=usr/doc/squeak/LICENSE.gz
+file path=usr/doc/squeak/README.Contributing.gz
+file path=usr/doc/squeak/README.Keyboard.gz
+file path=usr/doc/squeak/README.Sound.gz
+file path=usr/lib/$(MACH64)/squeak/$(PLUGIN_REV)/B3DAcceleratorPlugin.so
+file path=usr/lib/$(MACH64)/squeak/$(PLUGIN_REV)/FileAttributesPlugin.so
```

Comment:
You seem to have edited the sample-manifest.p5m. That prohibits its intention. If you run gmake sample-manifest again it will create a different file and thus it looks like something has to be change in squeak5.p5m.
So, generate this file and leave it untouched. Do all necessary changes in the squeak5.p5m (as you already did). So it will be possible later to check for real changes by comparing the old and the new sample-manifest.p5m. Unfortunately this is not perfect as you can see with your $(PLUGIN_REV) variable use.

Reply:
Ok, I'll regenerate the sample-manifest file, but I'll do that for a new version , say 5.0.7.
It is a good time to create 5.0.7 because I can merge new changes from opensmalltalk-vm into a new package (so the new package will effectively be a more recent squeakvm).  The build process will be the same,  there will be a .tar.gz for the 5.0.7 so it will then build from the tar.gz for the 5.0.7 release.   Finally I'd like to add a few words about "pharo".   Pharo VM compiles fine ;  it works.     I tried with "Pharo8.0-SNAPSHOT-64bit-a153e04.image" .

Reply:
Have you tried something that uses their new UFFI? The image relies heavily on it as far as I remember.

Reply:
No but I can imagine that that would perhaps not work.  I haven't tried it.   But we have to start somewhere ... It is a complex issue because on the one hand there is the VM (virtual machine) and on the other hand there's the development of the Smalltalk "image" which is let's call it "application level programming".